pkg/reconciler/broker/broker_test.go

```diff
@@ -218,8 +218,9 @@ func TestReconcile(t *testing.T) {
 	}}
 
 	brokerConfigs := map[string]*duckv1.KReference{
-		"rabbitmqClusterConfig": rabbitmqClusterConfigRef(),
-		"rabbitmqBrokerConfig":  rabbitmqBrokerConfigRef(),
+		"rabbitmqClusterConfig":     rabbitmqClusterConfigRef(),
+		"rabbitmqClusterConfigNoNS": rabbitmqClusterConfigRefNoNS(),
```

Comment:
This might be a bit overkill. The brokerConfigs get used to build out a matrix of tests for each of the broker configurations. The two that we support are rabbitmqBrokerConfig and the rabbitmqCluster.